src/mutable_arithmetics.jl

```diff
@@ -1,3 +1,183 @@
+###
+### Basic MA operations
+###
+
+MA.mutability(::Type{<:DGAE}) = MA.IsMutable()

```

Comment:
So you added this line and then you got `MethodError`s and implemented the ones below until you did not get any `MethodError` in the tests ?
If that's the case, it's great, that's how MA is supposed to work. Once you implement `mutability` then the code should not silently be slow, it should be fast or throw a `MethodError`.